app/views/users/shared/_otp_make_default_number.html.erb

```diff
@@ -12,12 +12,12 @@
       <%= t('two_factor_authentication.otp_make_default_number.instruction') %>
     </p>
     <label class="btn-border col-8">
-      <div class="checkbox">
+      <span class="checkbox">
```

Comment:
There's one more of these here which is still `div`, does it need to be updated as well?

https://github.com/18F/identity-idp/blob/9d07eb0d29de55318e3ae4d9e457b2543ff76b88/app/views/users/edit_phone/_make_default_number.html.erb#L10

Reply:
Must have missed one. Thanks for pointing this out!